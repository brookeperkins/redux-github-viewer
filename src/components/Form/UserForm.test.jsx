import React from 'react';
import { render, cleanup } from '@testing-library/react';
import UserForm from './UserForm';
import { Provider } from 'react-redux';
import store from '../../store';

describe('UserForm component', () => {
  afterEach(() => cleanup());
  it('renders UserForm', () => {
    const { asFragment } = render(
      <Provider store={store}>
        <UserForm />
      </Provider>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
