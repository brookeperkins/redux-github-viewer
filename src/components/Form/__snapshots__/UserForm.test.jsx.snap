// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserForm component renders UserForm 1`] = `
<DocumentFragment>
  <form>
    <label>
      Username:
      <textarea
        placeholder="username"
      />
    </label>
    <button>
      Submit
    </button>
  </form>
</DocumentFragment>
`;
